### PR TITLE
Fix trust secret

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

**Review by gtully — APPROVED** [review]
seems sensible to require a .pem format as the other supported keys are already locked down.

**Comment by gtully** [comment]
btw, nice use of the reason string to capture the error in the status

File: pkg/utils/certutil/certutil.go [hunk 1]
```diff
@@ -160,14 +160,17 @@ func isSecretFromBundle(secret *corev1.Secret) bool {
...
-	//extract the key of the secret's only entry
-	bundleName := ""
+func getBundleNameFromSecret(secret *corev1.Secret) (string, error) {
+	//extract the bundle target secret key that ends with .pem
+	//the bundle target secret could include keys for additional formats jks/pkcs12
 	for key := range secret.Data {
-		bundleName = key
-		break
+		//the bundle target secret key must ends with .pem
+		if strings.HasSuffix(key, ".pem") {
```
Comment (gaohoward) [inline pkg/utils/certutil/certutil.go:168]: Maybe add a note in the doc?

**Review by gaohoward — APPROVED** [review]

File: pkg/utils/certutil/certutil.go (hunk 1 above)
Comment (brusdev) [reply pkg/utils/certutil/certutil.go:168]: I added a note.

**Pull request merged**